docs/content/concepts/assets/graph-backed-assets.mdx

```diff
@@ -1,11 +1,11 @@
 ---
 title: Graph-Backed Assets | Dagster
-description: Defining a software-defined asset with multiple discrete computations combined in a graph.
+description: Defining a software-defined asset with multiple discrete computations combined in an op graph.
 ---
 
 # Graph-Backed Assets
 
-[Basic software-defined assets](/concepts/assets/software-defined-assets#a-basic-software-defined-asset) are computed using a single op. If generating an asset involves multiple discrete computations, you can use graph-backed assets by separating each computation into an op and building a graph to combine your computations. This allows you to launch re-executions of runs at the op boundaries, but doesn't require you to link each intermediate value to an asset in persistent storage.
+[Basic software-defined assets](/concepts/assets/software-defined-assets#a-basic-software-defined-asset) are computed using a single op. If generating an asset involves multiple discrete computations, you can use graph-backed assets by separating each computation into an op and building an op graph to combine your computations. This allows you to launch re-executions of runs at the op boundaries, but doesn't require you to link each intermediate value to an asset in persistent storage.
```

Comment:
"separating each computation into an op and assembling them into an op graph to combine your computations"

I think is more clear

Reply:
👍